File: 11.Sorting/python/1.bubble-sort.py
arr = [2, 5, 6, 3, 9, 5, 8, 7]


def bubbleSort(arr):
    length = len(arr) - 1
    for i in range(0, length):
        for j in range(0, length):
            if arr[j] > arr[j + 1]:
                temp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = temp
        length -= 1
    return arr


sortedArray = bubbleSort(arr)
print(sortedArray)

File: 11.Sorting/js/1.bubble-sort.js
const arr = [2, 5, 6, 3, 9, 5, 8, 7]

function bubbleSort(arr) {
  let len = arr.length - 1
  for (let i = 0; i < len; i++) {
    for (let j = 0; j < len; j++) {
      if (arr[j] > arr[j + 1]) {
        const temp = arr[j]
        arr[j] = arr[j + 1]
        arr[j + 1] = temp
      }
    }
    len--
  }
  return arr
}

const sortedArray = bubbleSort(arr)
console.log(sortedArray)